tests/components/plex/mock_classes.py

```diff
@@ -116,6 +118,16 @@ def __init__(
 
         self._systemAccounts = list(map(MockPlexSystemAccount, range(num_users)))
 
+        self._library = MockPlexLibrary(self)
+
+        self._playlists = [
```

Comment:
Not sure how expensive this is, but would it make sense to lazy initialize a bunch of these things so you don't have a big overhead on each test that uses it?

Reply:
Done. Makes Plex tests ~3% faster 😄